changelog.txt

```diff
@@ -18,6 +18,9 @@ Template for new versions:
 
 # Future
 
+## Structures
+- updated codegen to generate enum trait constants as ``consteval``
```

Comment:
"consteval"?

Reply:
oops - i was experimenting with making something else `consteval` and apparently had that on the brain when i wrote that